osgp/platform/osgp-secret-management/src/main/webapp/META-INF/context.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor 
+  license agreements. See the NOTICE file distributed with this work for additional 
+  information regarding copyright ownership. The ASF licenses this file to 
+  You under the Apache License, Version 2.0 (the "License"); you may not use 
+  this file except in compliance with the License. You may obtain a copy of 
+  the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required 
+  by applicable law or agreed to in writing, software distributed under the 
+  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS 
+  OF ANY KIND, either express or implied. See the License for the specific 
+  language governing permissions and limitations under the License. -->
+<!-- The contents of this file will be loaded for each web application -->
+<Context useHttpOnly="true">
+
+  <!-- Default set of monitored resources -->
+  <WatchedResource>WEB-INF/web.xml</WatchedResource>
+
+  <!-- App config files -->
+  <Environment name="osgp/Global/config" value="/etc/osp/global.properties"
+               type="java.lang.String" override="true"/>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=xml%3AIndentCheck)